prebid-server/hosting/bidder-specific-guidelines.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: page_v2
+title: Bidder-Specific Guidelines for PBS Host Companies
```

Comment:
The html template isn't welcoming to long titles. How do you feel about "Bidder-Specific Hosting Guidelines" to keep it on one line?